README.md

```diff
@@ -427,3 +458,8 @@ caching, and rate limiting.
 
 ### Sorting fields
 Sorting fields include a compound index on on the filed to sort and a unique field. Because in cases where there were large amounts of data that had the same value that was being evaluated for sort, the was not a stable sort view for results and the results users received were inconsistent, some records given more than once, others given multiple times.
+
+### Database mirrors/replicas
+Database mirrors/replicas are supported by the API if the `SQLA_FOLLOWERS` is set to one or more valid connection strings.  By default, setting this environment variable will shift all `read` operations to any mirrors/replicas that are available (and randomly choose one to target per request if there are more than one).
+
+You can optionally choose to restrict traffic that goes to the mirrors/replicas to be the asynchronous tasks only by setting the `SQLA_RESTRICT_FOLLOWER_TRAFFIC_TO_TASKS` environment variable to something that will evaluate to `True` in Python (simply using `True` as the value is fine).  If you do this, you can also restrict which tasks are supported on the mirrors/replicas.  Supported tasks [are configured here](https://github.com/18F/openFEC/blob/feature/modify-mirror-support/webservices/rest.py#L69) by adding their fully qualified names to the list in order to whitelist them.  By default, only the `download` task is enabled.
```

Comment:
Looks like the relevant line might be 71, but this is a brittle way to describe the relevant config setting--how about referring to the name instead?